index.html

```diff
@@ -331,6 +331,133 @@ <h2 class="contacts__title">Just Say Hello!</h2>
+                                <img class="blog__readmore" src="images/Skills/icons/arrow-white.svg" alt="Arrow"/>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="swiper-slide blog__box">
+                        <div class="blog__image__holder">
+                            <picture>
+                                <source srcset="images/Blog/Mobile/shoe.svg" media="(max-width: 760px)">
+                                <source srcset="images/Blog/Tablet/shoe.svg" media="(max-width: 1024px)">
+                                <img class="blog__image" src="images/Blog/Desktop/shoe.svg" alt="Blog Image">
+                            </picture>
+                            <p class="blog__date">25 July 2020</p>
+                        </div>
+                        <div class="blog__content">
+                            <div class="blog__interactions">
+                                <p class="interaction__text">25 comments</p>
+                                <p class="interaction__text">17 Likes</p>
+                                <p class="interaction__text">By Adams</p>
+                            </div>
+                            <h2 class="blog__title">Design is not just color it speaks visually...</h2>
+                            <div class="blog__more">
+                                <p class="blog__readmore">Read more</p>
+                                <img class="blog__readmore" src="images/Skills/icons/arrow-white.svg" alt="Arrow"/>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="swiper-slide blog__box">
+                        <div>
+                            <picture>
+                                <source srcset="images/Blog/Mobile/bird.svg" media="(max-width: 760px)">
+                                <source srcset="images/Blog/Tablet/bird.svg" media="(max-width: 1024px)">
+                                <img class="blog__image" src="images/Blog/Desktop/bird.svg" alt="Blog Image">
+                            </picture>
+                            <p class="blog__date">27 July 2020</p>
+                        </div>
+                        <div class="blog__content">
+                            <div class="blog__interactions">
+                                <p class="interaction__text">25 comments</p>
+                                <p class="interaction__text">17 Likes</p>
+                                <p class="interaction__text">By Adams</p>
+                            </div>
+                            <h2 class="blog__title">Design is not just color it speaks visually...</h2>
+                            <div class="blog__more">
+                                <p class="blog__readmore">Read more</p>
+                                <img class="blog__readmore" src="images/Skills/icons/arrow-white.svg" alt="Arrow"/>
```

Comment:
the arrow should be smaller

Reply:
The arrow is the same size for me in both my project and Figma.